packages/dialog/src/vaadin-dialog.js

```diff
@@ -40,7 +40,7 @@ let memoizedTemplate;
  * @extends OverlayElement
  * @private
  */
-class DialogOverlay extends mixinBehaviors(IronResizableBehavior, OverlayElement) {
+export class DialogOverlay extends mixinBehaviors(IronResizableBehavior, OverlayElement) {
```

Comment:
Could add this to the declaration file also:

<img width="939" alt="Screenshot 2021-11-25 at 16 08 51" src="https://user-images.githubusercontent.com/1222264/143456097-c83203d0-de9c-42e3-9b38-9df1bbb0b952.png">

Reply:
Thanks, added.